esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +215,38 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    @unittest.skipIf('OS_FUNCTIONAL_NODE_[...]' not in os.environ.keys(),
+                     "env variable to test node list function is not set")
+    def test_node_list_long(self):
```

Comment:
This test's logic should be updated to account for the fact that either environment variable can be set.  (see: https://github.com/CCI-MOC/python-esileapclient/pull/47#discussion_r1114672728)